README.md

```diff
@@ -97,6 +100,12 @@ always welcome.  Reports with patches are even more welcome.
 Additional Information
 ----------------------
 
-Up to date information about Octave is available on the WWW at
+Up to date information about Octave is available at
 <https://www.octave.org>, or ask for help via email
 <help@octave.org>.
+
+Copyright (C) 1996-2024 The Octave Project Developers
+
+See the file 
+[`COPYRIGHT.md`](https://github.com/gnu-octave/octave/blob/default/COPYRIGHT.md)
```

Comment:
Same as above:

The GitHub mirror is just that (a mirror of the main repository).
We shouldn't hard-code paths to the mirror repository here.